components/core/JsInterop/interop.ts

```diff
@@ -3,10 +3,20 @@ export function getDom(element) {
         element = document.body;
     } else if (typeof element === 'string') {
         element = document.querySelector(element);
+    } else {
+        element = null;
```

Comment:
Why null? If you pass in an ElementReference from C#,js should get the dom.

Reply:
sorry, I forgot it.